releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,14 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 5.3.0-preview - July 27, 2022
+* Added the following new API for the App Configuration geo-replication support.
+
+```cs
+public AzureAppConfigurationOptions Connect(IEnumerable<Uri> endpoints, TokenCredential credential)
+```
+
+The new API allows you to provide an ordered list of replica endpoints of your App Configuration store. The App Configuration provider will fail over to other replicas when a replica is not accessible (either the service is unavailable or there are networking issues). A sample application can be found [here](https://github.com/Azure/AppConfiguration-DotnetProvider/tree/release/preview/v5/examples/ConsoleAppWithFailOver).
```

Comment:
NIT: add two spaces in front, so it aligns with the bullet point.

Reply:
Done!